src/main/java/com/epam/reportportal/service/impl/ElasticMigrationServiceImpl.java

```diff
@@ -90,26 +90,26 @@ private void migrateLogsBeforeDate(LocalDateTime date) {
 				Map.of("time", date),
 				new LogRowMapper()
 		);
-		elasticSearchClient.save(createIndexMap(logMessageWithLaunchIdList, logMessageWithoutLaunchIdList));
+		elasticSearchClient.save(createProjectMap(logMessageWithLaunchIdList, logMessageWithoutLaunchIdList));
 		LOGGER.info("Migration completed at {}", LocalDateTime.now());
 	}
 
-	private TreeMap<Long, List<LogMessage>> createIndexMap(List<LogMessage> logsWithLaunchId, List<LogMessage> logsWithoutLaunchId) {

```

Comment:
Better use more clear name - groupLogByProject for instance or somethings, that reflect method action.